apps/src/javalab/CommitDialog.jsx

```diff
@@ -151,6 +191,9 @@ const styles = {
     fontFamily: '"Gotham 5r", sans-serif',
     color: color.dark_charcoal
   },
+  footerButtons: {
```

Comment:
Is this used?

Reply:
Cleanup crew over here catching what I missed! Thank you!